mlflow/models/resources.py

```diff
@@ -217,6 +218,29 @@ def from_dict(cls, data: Dict[str, str]):
         return cls(genie_space_id=data["name"])
 
 
+class DatabricksTable(DatabricksResource):
+    """
+    Define a Databricks Table to serve a model.
```

Comment:
What is "Databricks Table"? Is it an UC table, and how the model access to that? Please leave some context on what this resource is and why we track it.